http-server/src/test/java/org/triplea/server/lobby/game/listing/GameListingTest.java

```diff
@@ -108,48 +104,42 @@ void getGames() {
   final class KeepAlive {
     @Test
     void noGamesPresent() {
-      when(cache.getIfPresent(ID_0)).thenReturn(null);
-
       final boolean result = gameListing.keepAlive(API_KEY_0, GAME_ID_0);
       assertThat("Game not found, keep alive should return false", result, is(false));
 
-      verify(cache, never()).put(any(), any());
+      assertThat(cache.asMap(), is(Map.of()));
     }
 
     @Test
     void gameExists() {
-      when(cache.getIfPresent(ID_0)).thenReturn(lobbyGame0);
+      cache.put(ID_0, lobbyGame0);
 
       final boolean result = gameListing.keepAlive(API_KEY_0, GAME_ID_0);
 
       assertThat("Game found, keep alive should return true", result, is(true));
-      verify(cache).put(ID_0, lobbyGame0);
+      assertThat(cache.asMap(), is(Map.of(ID_0, lobbyGame0)));
```

Comment:
I'm not sure if this assertion is testing very much, seems like it should be true based on the library spec of the cache we are using and not anything related to the `keepAlive`. The call to `put` is important for `keepAlive` so that it extends the cache expiration. 

I suspect in order to maintain the same level of test, we either need a mock of the cache, or we will need to extract the cache implementation behind an API (and then inject+mock that):

This same comment/concern follows for the rest of the examples in this test